pessimistic-transaction.md

```diff
@@ -111,7 +111,11 @@ Pessimistic transactions in TiDB behave similarly to those in MySQL. See the min
 
 2. TiDB does not support `SELECT LOCK IN SHARE MODE`.
 
-    When `SELECT LOCK IN SHARE MODE` is executed, it has the same effect as that without the lock, so the read or write operation of other transactions is not blocked.
+    TiDB does not support `SELECT LOCK IN SHARE MODE` by default.
+
+    You can enable [`tidb_enable_noop_functions`](/system-variables.md#tidb_enable_noop_functions-new-in-v40) to make TiDB compatible with the `SELECT LOCK IN SHARE MODE` statement. The effect of the statement is the same as when there is no lock, and it will not block the read and write operations of other transactions.
```

Comment:
```suggestion
    TiDB does not support `SELECT LOCK IN SHARE MODE` by default. You can enable [`tidb_enable_noop_functions`](/system-variables.md#tidb_enable_noop_functions-new-in-v40) to make TiDB compatible with the `SELECT LOCK IN SHARE MODE` statement. The effect of the statement is the same as when there is no lock, and it will not block the read and write operations of other transactions.
```